SECURITY.md

```diff
@@ -0,0 +1,95 @@
+- Only use exploits to the extent necessary to confirm a vulnerability’s presence. Do not use an exploit to compromise or exfiltrate data, establish persistent command line access, or use the exploit to pivot to other systems.
+- Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.
+- Do not submit a high volume of low-quality reports.
+
+Once you’ve established that a vulnerability exists or encounter any sensitive data (including personally identifiable information, financial information, or proprietary information or trade secrets of any party), **you must stop your test, notify us immediately, and not disclose this data to anyone else.**
+
+
+## Test methods
+
+The following test methods are not authorized:
+
+- Network denial of service (DoS or DDoS) tests or other tests that impair access to or damage a system or data
+- Physical testing (e.g. office access, open doors, tailgating), social engineering (e.g. phishing, vishing), or any other non-technical vulnerability testing
+
+## Scope
+
+This policy applies to the following systems and services:
+- immudb database core application, 
+- immuadmin and immuclient applications
+- Golang Immudb SDK
+Source code of the above tools [is available on GH](https://github.com/codenotary/immudb).
+Also:
+- Java Immudb SDK ([repo link](https://github.com/codenotary/immudb4j))
+- Python Immudb SDK ([repo link](https://github.com/codenotary/immudb-py))
+- .Net Immudb SDK ([repo link](https://github.com/codenotary/immudb4dotnet))
+
+*To decide:*
+- Ruby on rails SDK ([repo link](https://github.com/ankane/immudb-ruby))
+- Nodejs SDK ([repo link](https://github.com/codenotary/immudb-node))
+
+
+Any service not expressly listed above, such as any connected services, are excluded from scope and are not authorized for testing. Additionally, vulnerabilities found in systems from our vendors fall outside of this policy’s scope and should be reported directly to the vendor according to their disclosure policy (if any). If you aren’t sure whether a system is in scope or not, contact us at (*enter email address here*) before starting your research (or at the security contact for the system’s domain name listed in the .gov WHOIS).
+
+Though we develop and maintain other internet-accessible systems or services, we ask that active research and testing only be conducted on the systems and services covered by the scope of this document. If there is a particular system not in scope that you think merits testing, please contact us to discuss it first. We will increase the scope of this policy over time.
+
+## Reporting a vulnerability
+
+We accept vulnerability reports via (*enter email address or other way*). Reports may be submitted anonymously. If you share contact information, we will acknowledge receipt of your report within 3 business days.

```

Comment:
Leaving comment so we don't forget to fill in the email address.

Reply:
Added email in the next commit.